certification/pyxis/submit.go

```diff
@@ -51,31 +51,30 @@ func (p *pyxisClient) SubmitResults(ctx context.Context, certInput *certificatio
 
 	// Create the image, or get it if it already exists.
 	certImage, err = p.createImage(ctx, certImage)
-	if err != nil && err != errors.ErrPyxis409StatusCode {
-		log.Error(err, "could not create image")
-		return nil, err
-	}
-
 	if err != nil && err == errors.ErrPyxis409StatusCode {
 		certImage, err = p.getImage(ctx, originalImageDigest)
 		if err != nil {
-			log.Error(err, "could not get image")
+			log.Error(fmt.Errorf("%w: could not get image", err))
 			return nil, err
 		}
 	}
+	if err != nil && err != errors.ErrPyxis409StatusCode {
+		log.Error(fmt.Errorf("%w: could not create image", err))
+		return nil, err
+	}
```

Comment:
I know we (and Go) typically recommends keeping the flow as close to the left-most indentation as possible (as in, nest less), but this seems like a prime opportunity where nesting would fit. 

```Go
if err != nil {
    if err == errors.ErrPyxis409StatusCode { // or a case statement if we want to match more than one type of error
        ...
    }

    log...
    return...
}

Reply:
This change actually wasn't supposed to be here. :) Only the log fix. I had re-ordered it. I'll play with it.